server/scsynth/SC_MiscCmds.cpp

```diff
@@ -428,90 +428,90 @@ SCErr meth_n_mapan(World *inWorld, int inSize, char *inData, ReplyAddress* /*inR
 SCErr meth_n_set(World *inWorld, int inSize, char *inData, ReplyAddress *inReply);
 SCErr meth_n_set(World *inWorld, int inSize, char *inData, ReplyAddress* /*inReply*/)
 {
-    sc_msg_iter msg(inSize, inData);
-    Node *node = Msg_GetNode(inWorld, msg);
-    if (!node) return kSCErr_NodeNotFound;
-    while( msg.remain()>=8) {
-	    int i = 0;
-	    int loop = 0;
-	    if (msg.nextTag('i') == 's') {
-		int32* name = msg.gets4();
-		int32 hash = Hash(name);
-		do {
-			switch (msg.nextTag('f') ) {
-			case  'f' :
-			case  'i' :
-			{
-				float32 value = msg.getf();
-				Node_SetControl(node, hash, name, i, value);
-				++i;
-				break;
-			}
-			case 's' :
-			{
-				const char* string = msg.gets();
-				if ( *string == 'c') {
-					int bus = sc_atoi(string+1);
-					Node_MapControl(node, hash, name, i, bus);
-					++i;
+	sc_msg_iter msg(inSize, inData);
+	Node *node = Msg_GetNode(inWorld, msg);
+	if (!node) return kSCErr_NodeNotFound;
+	while( msg.remain()>=8) {
+		int i = 0;
+		int loop = 0;
+		if (msg.nextTag('i') == 's') {
+			int32* name = msg.gets4();
+			int32 hash = Hash(name);
+			do {
+				switch (msg.nextTag('f') ) {
+					case  'f' :
+					case  'i' :
+					{
+						float32 value = msg.getf();
+						Node_SetControl(node, hash, name, i, value);
+						++i;
+						break;
+					}
+					case 's' :
+					{
+						const char* string = msg.gets();
+						if ( *string == 'c') {
+							int bus = sc_atoi(string+1);
+							Node_MapControl(node, hash, name, i, bus);
+							++i;
+						}
+						if ( *string == 'a') {
+							int bus = sc_atoi(string+1);
+							Node_MapAudioControl(node, hash, name, i, bus);
+							++i;
+						}
+						break;
+					}
+					case ']':
+						msg.count++;
+						loop--;
+						break;
+					case '[':
+						msg.count++;
+						loop++;
+						break;
 				}
-			    if ( *string == 'a') {
-				int bus = sc_atoi(string+1);
-				Node_MapAudioControl(node, hash, name, i, bus);
-				++i;
-			    }
-			    break;
 			}
-			case ']':
-			    msg.count++;
-			    loop--;
-			    break;
-			case '[':
-			    msg.count++;
-			    loop++;
-			    break;
-		    }
-		}
-		while (loop);
-	    } else {
-		int32 index = msg.geti();
-		do {
-		    switch (msg.nextTag('f') ) {
-			case  'f' :
-			case  'i' :
-			{
-				float32 value = msg.getf();
-				Node_SetControl(node, index + i, value);
-				++i;
-				break;
-			}
-			case 's' :
-			{
-				const char* string = msg.gets();
-				if ( *string == 'c') {
-				    int bus = sc_atoi(string+1);
-				    Node_MapControl(node, index + i, bus);
-				    ++i;
-				}
-				if ( *string == 'a') {
-				    int bus = sc_atoi(string+1);
-				    Node_MapAudioControl(node, index + i, bus);
-				    ++i;
+			while (loop);
```

Comment:
Style thing--would you consider moving the `while`s to the same line as the close-brace? it's easy to mistake for a no-op, standalone `while` otherwise.